src/content/docs/fr/reference/errors/invalid-content-entry-frontmatter-error.mdx

```diff
@@ -10,7 +10,7 @@ githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/
 "date" doit être une date valide.
 
 ## Qu'est-ce qui a mal tourné ?
-Une entrée Markdown ou MDX dans `src/content/` ne correspond pas au schéma de sa collection.
+Une entrée Markdown ou MDX ne correspond pas au schéma de sa collection.
 Vérifiez que tous les champs requis sont présents et que tous les champs sont du bon type.
-Vous pouvez vérifier par rapport au schéma de collection dans votre fichier `src/content/config.*`.
+Vous pouvez vérifier par rapport au schéma de collection dans votre fichier `src/content.config.*`.
```

Comment:
```suggestion
Vous pouvez vérifier par rapport au schéma de collection dans votre fichier `src/content/config.*`.
```

Reply:
Hum... https://docs.astro.build/en/guides/upgrade-to/v5/#updating-existing-collections 😅 
Since v5 the new config file format is `src/content.config.ts`, the `src/content` directory is no longer required. So I think this is not an error (and this matches the change https://github.com/withastro/docs/pull/9240/files#diff-bdfea8d13913bc35d9cf780c1c35064c6355a98948c504ccbfc55c392aed8c6a ).

Reply:
Oooh okay thanks for the clarification!